.prettierrc

```diff
@@ -18,7 +18,17 @@
       "files": "*.md",
       "options": {
         "printWidth": 100,
-        "tabWidth": 2,
+        "tabWidth": 4,
```

Comment:
I put 2 here in the past so that our docs are more readable. We can ignore it for this repo if you want. Not sure if we should do it for the other repos as well that contain docs.

Feel free to ignore this comment

Reply:
I added 4 because in vscode you can only set it for all files. Thus when you use `tab` while writing you get 4 and prettier makes then 2 out of it.

Reply:
Please create a task to change it in other repos too